spec/basic/elements_spec.js

```diff
@@ -470,15 +470,13 @@ describe('ElementArrayFinder', function() {
     browser.get('index.html#/form');
     var labels = element.all(by.css('#animals ul li')).map(function(elm) {
       return {
-        text: elm.getText(),
-        inner: elm.getInnerHtml()
+        text: elm.getText()
```

Comment:
We should have something else here to make sure multiple promises are waited for. How about `elm.getTagName`?